Bio-CS08-F19-Xingxing-G.txt

```diff
@@ -0,0 +1 @@
+Hi, Everyone! This is Xingxing, Geng(she/her/hers) and I am a fourth year CS major student. I will mentor CS8 this quarter and this will be my first time to be tutor. Recalling my first year in UCSB, when I took CS8, it is my first to program, to write codes. At first, I was lost in class and confused about the materials that the professor talked about. Fortunately, TAs and tutors in labs were patient and knowledgeable. They helped me a lot and answered questions properly. So, after three years learning Computer Science, I want to apply what I learned to practice and help students who may struggle to start. I will be in the closed lab and open lab time and this will be a good time to ask me questions. And I will also attend the class, so please feel free to come to me. I will be very happy to talk with you about any questions or worries.  For CS8, I would say you can learn many useful programming tools in this class even though you are not CS major students, CS8 will benefit you a lot. Python and programming skills will be taught in this class and those contribute to your future studies. In order to succeed in this course, I would recommend you always start your lab assignments early and come to lab with your questions. And starting your assignment by carefully reading instructions. Since CS8 is an introduction course, its assignment instructions are clear and straight forward. You can find most of your answers in instructions. Additionally, never wait to the last minute to begin your work because people are easily going to be panic when they run out of time. This will cause you in trouble and make your work unsatisfactory! One last thing I want mention is that I am currently working on research project. If you are interested in doing research or eager to pursue your career in computer science, I will be also glad to talk with you and see what we can do. I hope you can enjoy yourselves in this course and please always feel free to contact with me! 
```

Comment:
Can you add this information to your bio in the `_info/staff` section of the website?

Once you do, you can delete this text file.

Reply:
I have moved my picture to info/mentorphotos directory and add my bio into info/staff.md file. I also deleted my bio text file.